doc/glossary.rst

```diff
@@ -1479,8 +1479,9 @@ functions or non-estimator constructors.
 
         ``n_jobs`` is an int, specifying the maximum number of concurrently
         running jobs.  If set to -1, all CPUs are used. If 1 is given, no
-        parallel computing code is used at all.  For n_jobs below -1, (n_cpus +
-        1 + n_jobs) are used. Thus for n_jobs = -2, all CPUs but one are used.
+        job-level parallel computing code is used at all, which is useful for
```

Comment:
Maybe "no joblib level parallelism is used at all".
I also think it's important to emphasize that other parallelism may be used as mentionned above with a link to the FAQ, as a number of users don't seem to be aware of it.

Possibly,
"""
Even with n_jobs = 1, parallelism may occur be due to numerical processing libraries (see FAQ#section_on_parallelism).
"""
?